docs/openapi/components/schemas/credentials/SIMASteelImportLicenseApplicationCredential.yml

```diff
@@ -37,24 +37,15 @@ properties:
         - SIMASteelImportLicenseApplicationCredential
   id:
     type: string
-  name:
```

Comment:
name should be in the example, but not required. "SIMA Steel Import License Application"